File: DESIGN-CSharp/Strategy/FlightStrategy.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Strategy
{
    public abstract class FlightStrategy
    {
        public abstract void execute();
    }

    public class NormalAttack : FlightStrategy
    {
        public override void execute()
        {
            Console.WriteLine("execute normal attack ");
        }
    }

    public class UseSkill : FlightStrategy
    {
        public override void execute()
        {
            Console.WriteLine("execute skill attack ");
        }
    }

    public class UseItem : FlightStrategy
    {
        public override void execute()
        {
            Console.WriteLine("execute item attack ");
        }
    }
}

File: BehavioralPatterns/Strategy/FlightStrategy.py
class FlightStrategy:
    def execute(self):
        pass

class NormalAttack(FlightStrategy):
    def execute(self):
        print("execute normal attack ")

class UseSkill(FlightStrategy):
    def execute(self):
        print("execute skill attack")

class UseItem(FlightStrategy):
    def execute(self):
        print("execute item attack")